sklearn/model_selection/_validation.py

```diff
@@ -474,6 +481,14 @@ def _fit_and_predict(estimator, X, y, train, test, verbose, fit_params,
         estimator.fit(X_train, y_train, **fit_params)
     func = getattr(estimator, method)
     predictions = func(X_test)
+    if method in ['decision_function', 'predict_proba', 'predict_log_proba']:
+        true_classes = np.unique(y)
```

Comment:
Why not have `_, n_classes =  np.unique(y, return_counts=True)` as you don't use the "true_classes"?